crates/editor/src/editor.rs

```diff
@@ -9287,8 +9281,8 @@ impl Editor {
                     self.change_selections(Some(Autoscroll::fit()), cx, |s| {
                         s.select(vec![Selection {
                             id: selection.id,
-                            start: primary_range.start,
-                            end: primary_range.start,
+                            start: primary_range.start.to_offset(&buffer),
```

Comment:
Nit: we can hoist out the result of this call to avoid calling `to_offset` twice here.